DDDEastAnglia.Tests/Helpers/Email/SendGrid/HtmlRendererFacts.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DDDEastAnglia.Helpers.Email.SendGrid;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Helpers.Email.SendGrid
+{
+    [TestFixture]
+    public class HtmlRendererFacts
+    {
+        [TestFixture]
+        public class Render_Should
+        {
+            [Test]
+            public void Convert_Markdown_To_HTML()
+            {
+                var sut = CreateSut();
```

Comment:
I find calling the variable `sut` and the method `CreateSut()` distracts me when reading the tests. Calling the variable `htmlRenderer` would help greatly, even if the method wasn't called `CreateRenderer()`..